.github/workflows/test.yml

```
@@ -258,6 +263,40 @@ jobs:
           cd prusti-contracts/prusti-contracts-test/
           cargo +stable build
 
+  release-build:
```

Avoid running the job if any of the quick checks fails.
```suggestion
  release-build:
    needs: [fmt-check, clippy-check, check-deps, smir-check, quick-tests]
```